superset/views/core.py

```diff
@@ -837,7 +837,7 @@ def explore(  # pylint: disable=too-many-locals,too-many-return-statements,too-m
                 bootstrap_data, default=utils.pessimistic_json_iso_dttm_ser
             ),
             entry="explore",
-            title=title,
+            title=title.__str__(),
```

Comment:
wait, so `title` is a instance of LazyString? Seems like it should just be a string here

Reply:
Title is an instance of lazystring in certain cases. The call to `__str__()` evaluates it to a string.

Reply:
i see now, it's `title = _("Explore - %(table)s", table=table_name)` that's the problematic code path

Reply:
any reason to use `title.__str__()` over `str(title)`?